tests/model_validation/tests.py

```diff
@@ -1,20 +1,28 @@
 from django.core import management
-from django.core.checks import Error, run_checks
+from django.core.checks import Error
+from django.core.checks.model_checks import _check_lazy_references
 from django.db.models.signals import post_init
 from django.test import SimpleTestCase
-from django.test.utils import override_settings
+from django.test.utils import isolate_apps, override_settings
 from django.utils import six
 
 
 class OnPostInit(object):
     def __call__(self, **kwargs):
         pass
 
+    def on_post_init(self):
+        pass
+
 
 def on_post_init(**kwargs):
     pass
 
 
+def print_model(model):
```

Comment:
Mmm, this is a dummy function to test the "default" error case for a missing lazy operation – that is, one that isn't related to a signal or field. Its body and name are irrelevant – I'll update to make that clearer.